packages/devextreme-cli/src/templates/react/sample-pages/display-data/display-data.js

```diff
@@ -27,86 +26,84 @@ export default class extends React.Component {
       'Task_Completion',
       'ResponsibleEmployee/Employee_Full_Name'
     ]
-  };
+  }), []);
 
-  priorities = [
+  const priorities = useMemo(() => [
```

Comment:
Maybe instead of using `useMemo` we should better move such constants out of the component? (to the module level)